src/js/modules/graph.js

```diff
@@ -250,39 +252,28 @@ export function forcedGraph (svgElement, graph) {
     drawNodes(node)
   })
 
-  /*
-    function zoomFit(paddingPercent, transitionDuration) {
-        var bounds = g.node().getBBox();
-        var parent = g.node().parentElement;
-
-        var fullWidth = parent.clientWidth,
-            fullHeight = parent.clientHeight;
-        var width = bounds.width,
-            height = bounds.height;
-        var midX = bounds.x + width / 2,
-            midY = bounds.y + height / 2;
-
-        if (width == 0 || height == 0) return; // nothing to fit
-        var scale = (paddingPercent || 0.75) / Math.max(width / fullWidth, height / fullHeight);
-        var translate = [fullWidth / 2 - scale * midX, fullHeight / 2 - scale * midY];
-
-        console.trace("zoomFit", translate, scale);
-        // g
-        //     .transition()
-        //     .duration(transitionDuration || 0) // milliseconds
-        //     .call(zoomHandler.translate(translate).scale(scale).event);
-
-        var transform = d3.zoomIdentity
-            .translate(translate[0], translate[1])
-            .scale(scale);
-
-        g
-            .transition()
-            .duration(transitionDuration || 0) // milliseconds
-            .call(zoom.transform, transform);
-    }
-    zoomFit(.7, 1000)
-    */
+  function zoomFit (paddingPercent, transitionDuration) {
+    var bounds = svg.node().getBBox()
+    var parent = svg.node().parentElement
+
+    var fullWidth = parent.clientWidth
+    var fullHeight = parent.clientHeight
+    var width = bounds.width
+    var height = bounds.height
+    var midX = bounds.x + width / 2
+    var midY = bounds.y + height / 2
+    if (width === 0 || height === 0) return // nothing to fit
+    var scale = (paddingPercent || 0.75) / Math.max(width / fullWidth, height / fullHeight)
+    var translate = [fullWidth / 2 - scale * midX, fullHeight / 2 - scale * midY]
+    console.trace('zoomFit', translate, scale)
+
+    svg
+      .transition()
+      .duration(transitionDuration || 0) // milliseconds
+      .call(zoomHandler.translateTo, translate[0], translate[1])
```

Comment:
I'm only using `translateTo`.
For reference, I tried to use `scaleTo(scale)` but that does not produce the expected result.